woof-code/rootfs-skeleton/etc/rc.d/rc.shutdown

```diff
@@ -130,10 +136,26 @@ for i in /sys/class/net/* ; do
 	fi
 done
 
-#terminate dbus daemon
-killall dbus-daemon 2>/dev/null
+#=======================================================================
 
-killall udevd > /dev/null 2>&1
+# terminate stuff
+# dbus daemon
+if [ -e /var/run/dbus/dbus.pid ]; then
+	DPID=$(cat /var/run/dbus/dbus.pid)
+	kill $DPID >/dev/null 2>&1
+	rm -f /var/run/dbus/dbus.pid
+fi
+killall dbus-daemon >/dev/null 2>&1
```

Comment:
So after this line..
```
killall dbus-launch
```
@dimkr :question: 
I'm aware there is some redundancy in this block but if we can't kill it with an axe maybe the sledge hammer works.

Reply:
Yes, I think this is good enough.